src/ui/simulator/windows/cleaner.cpp

```diff
@@ -57,7 +57,7 @@ class CleaningThread final : public Yuni::Thread::IThread
         {
             std::lock_guard locker(pParent->pMutex);
             if (pParent->pInfos)
-                pParent->pInfos->onProgress.unbind();
+                pParent->pInfos->onProgress = nullptr;
```

Comment:
Need to check what happens if we call an unbinded Yuni::Bind function. Calling a nullptr std::function is UB (?)

Reply:
Done, will use `[]{}`, see https://stackoverflow.com/a/2982599